profiles/models.py

```diff
@@ -1,22 +1,23 @@
 from datetime import datetime
 
-from flask_sqlalchemy import SQLAlchemy
-
-db = SQLAlchemy()
+from profiles.database import db, UTCDateTime
 
 ID_LENGTH = 8
 
 
 class OrcidToken(db.Model):
     orcid = db.Column(db.String(19), primary_key=True)
     access_token = db.Column(db.String(255), unique=True, nullable=False)
-    expires_at = db.Column(db.DateTime(), nullable=False)
+    expires_at = db.Column(UTCDateTime, nullable=False)
```

Comment:
SQLAlchemy only seems to use naive datetimes, whereas we really want to make sure we talk UTC everywhere.